src/lightmap.cpp

```diff
@@ -331,6 +331,15 @@ void map::generate_lightmap()
                 }
             }
         }
+        // atomic light is always on
+        std::vector<int> light_indices = v->all_parts_with_feature(VPFLAG_ATOMIC_LIGHT);
```

Comment:
I don't know off hand either, but I would assume it will need to be updated wherever vehicle parts are added / removed / damaged. Maybe just checking on load in `savegame_json.cpp` would be enough?